### ISSUE TITLE
Save Euclidean

### ISSUE THREAD

**Issue by jvoight** [michaelmusty/BelyiDB#20]
Matt Radosevich's code allows us to compute many many Euclidean Belyi maps.

How do they get saved?  I only see the function `GammasToBelyiDB`; but I bet there's something somewhere in there, because we do display Euclidean Belyi maps in the LMFDB...

**Comment by SamSchiavone** [comment]
I think `GammasToBelyiDB` takes a sequence of triangle subgroups with all the Belyi map data computed to a BelyiDB object. You can then use `BelyiDBWrite` to write this BelyiDB object to file. (To read the file back in, there's `BelyiDBRead`.)

**Comment by jvoight** [comment]
Yep, that's right!  But we can't make a Gamma currently as a triangle in the Euclidean case.  
```
>> TriangleGroup(2,4,4);
                ^
Runtime error: Must be hyperbolic (for now, sorry!)

```

I'm loathe to add that because we build on GrpPSL2 (making GrpPSL2Tri), and in the Euclidean case, you don't have a subgroup of PSL_2(RR).  Maybe we can just rename?

**Comment by SamSchiavone** [comment]
Ah, I see. I think the intrinsic `BelyiDBAdhoc` on line 61 of `query.m` is how we dealt with spherical and Euclidean maps in the BelyiDB.
